input/forward.md

```diff
@@ -174,7 +174,7 @@ The default value `0` is to send RST rather than FIN to avoid lots of connection
 
 You can set positive value to send FIN on closing on non-Windows.
 
-(On Windows, Fluentd sends FIN when `linger_timeout` is `0` too).
+Note: On Windows, Fluentd sends FIN without depending on this setting.
 
```

Comment:
How about using `hint style='info' ` markup to emphasize it in updated explanation?

Reply:
Thanks! Great!
Done.